src/composable/composable.ts

```diff
@@ -202,8 +209,6 @@ export {
   all,
   collect,
   composable,
-  composable as cf,
-  composable as λ,
```

Comment:
I think `composable` is short enough and the lack of aliases creates a nice uniformity to code-bases. I'm not strongly opposed to reintroducing them however.